virt/virtual_machines/advanced_vm_management/virt-enabling-descheduler-evictions.adoc

```diff
@@ -7,12 +7,17 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 :FeatureName: Descheduler eviction for virtual machines
-include::snippets/technology-preview.adoc[]
 
-The xref:../../../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[descheduler] can be used to evict a running pod to allow the pod to be rescheduled onto a more suitable node. You must install the descheduler by using the {product-title} web console or OpenShift CLI (`oc`) before you can enable it on your virtual machine (VM).
+You can use the descheduler to evict pods so that the pods can be rescheduled onto more appropriate nodes. If the pod is a virtual machine, the pod eviction causes the virtual machine to be live migrated to another node.
+
+include::snippets/technology-preview.adoc[]
 
 include::modules/nodes-descheduler-profiles.adoc[leveloffset=+1]
 
 include::modules/nodes-descheduler-installing.adoc[leveloffset=+1]
 
 include::modules/virt-enabling-descheduler-evictions.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
```

Comment:
You also need to add a unique id after this line per our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies)